api/view/static/openapi.json

```diff
@@ -0,0 +1,298 @@
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/n3": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "application/n-triples": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/html": {
+                "schema": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/geometry/{geomId}": {
+      "get": {
+        "tags": [
+          "geometry"
+        ],
+        "summary": "Get geometry instance",
+        "description": "",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "text/plain": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "application/json": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/turtle": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "application/geo+json": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/html": {
+                "schema": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "operationId": "getGeometryByGeomId",
+        "parameters": [
+          {
+            "name": "geomId",
+            "in": "path",
+            "description": "Geometry ID",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "_view",
+            "in": "query",
+            "description": "View. If none selected, defaults to 'geometryview'.",
+            "required": false,
+            "schema": {
+              "type": "string",
+              "enum": [
+                "geometryview",
+                "centroid",
+                "simplifiedgeom",
+                "alternates"
+              ]
+            }
+          },
+          {
+            "name": "_format",
+            "in": "query",
+            "description": "Media type. If none selected, defaults to 'text/html' or whatever is in the Accept header.",
+            "required": false,
+            "schema": {
+              "type": "string",
+              "enum": [
+                "text/html",
+                "application/json",
+                "text/turtle",
+                "text/plain"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/search/latlng/{x},{y}": {
+      "get": {
+        "tags": [
+          "search"
+        ],
+        "summary": "Get geometry instances that contain specified position",
```

Comment:
"Get individual geometries that contain the specified position" (for consistency)